CHANGELOG.md

```diff
@@ -2,10 +2,81 @@
+- (Core) `confignet`: Mark module as Stable. ([#9801](https://github.com/open-telemetry/opentelemetry-collector/pull/9801))
+- (Core) `confmap/provider/envprovider`: Support default values when env var is empty ([#5228](https://github.com/open-telemetry/opentelemetry-collector/pull/5228))
+- (Core) `mdatagen`: mdatagen validateMetrics() support validate metrics in telemetry.metric ([#10925](https://github.com/open-telemetry/opentelemetry-collector/pull/10925))
+- (Core) `service/telemetry`: Mark useOtelWithSDKConfigurationForInternalTelemetry as stable ([#7532](https://github.com/open-telemetry/opentelemetry-collector/pull/7532))
+- (Core) `mdatagen`: Use cobra for the command, add version flag ([#11196](https://github.com/open-telemetry/opentelemetry-collector/pull/11196))
+
+- (Contrib) `processor/transform`: Add custom function to the transform processor to convert exponential histograms to explicit histograms. ([#33827](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33827))
+- (Contrib) `datadogconnector`: Map the new OTel semantic convention deployment.environment.name to env for OTLP traces in APM stats. The old convention deployment.environment still works. ([#35147](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35147))
+- (Contrib) `datadogexporter`: Map the new OTel semantic convention deployment.environment.name to env for OTLP traces, metrics and logs. The old convention deployment.environment still works. ([#35147](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35147))
+- (Contrib) `file_storage`: provide a new option to the user to create a directory on start ([#34939](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34939))
+- (Contrib) `headersetterextension`: adding default_value config. `default_value` config item applied in case context value is empty. ([#34412](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34412))
+- (Contrib) `kafkaexporter`: Add support for encoding extensions in the Kafka exporter. This change adds support for encoding extensions in the Kafka exporter. Loading extensions takes precedence over the internally supported encodings. ([#34384](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34384))
+- (Contrib) `datadogexporter`: Adds exporter.datadogexporter.metricremappingdisabled featuregate which disables renaming OpenTelemetry metrics to match Datadog semantics. This feature gate is only for internal use. ([#35025](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35025))
+- (Contrib) `otelarrowexporter`: Add BatcherConfig field following similar in OTLP exporter. ([#34802](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34802))
```

Comment:
I don't think we include any of these exporters. The list should filter out all components that are not part of our build.